modules/spark/src/test/scala/org/apache/ignite/spark/IgniteDataFrameSchemaSpec.scala

```diff
@@ -88,6 +111,24 @@ class IgniteDataFrameSchemaSpec extends AbstractDataFrameSpec {
 
         createPersonTable(client, DEFAULT_CACHE)
 
+        createMetaTestTable(client, DEFAULT_CACHE)
```

Comment:
Let's move this calls into the one test. 

I think the test should be the following:

1. Read `test` table. Check the initial fields list.
2. Execute `addColumnForTable`. Read `test` table. Check the fields list.
3. Execute `dropColumnForTable`. Read `test` table. Check the fields list.

Reply:
Fixed